extensions/functions_rounding.yaml

```
@@ -27,3 +27,52 @@ scalar_functions:
           - value: fp64
             name: "x"
         return: fp64
+  -
+    name: "round"
+    description: >
+      Rounding the value `x` to s decimal places.
```

```suggestion
      Rounding the value `x` to `s` decimal places.
```
This isn't markdown so I don't know if there is value in using backticks like this.  Do we do this elsewhere?  If we are using backticks consistently then `s` should also be quoted.